packages/@aws-cdk/aws-codepipeline/test/integ.pipeline-ecs-deploy.ts

```diff
@@ -0,0 +1,77 @@
+import ec2 = require('@aws-cdk/aws-ec2');
```

Comment:
The indentation in this file should be 2 spaces (I see it is in the other files, so this is probably a simple settings snafu).